server.js

```diff
@@ -1,32 +1,52 @@
-import express from 'express'
+import express, { response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import goldenGlobesData from './data/golden-globes.json'
+
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+
+// Add middlewares to ensable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/', (request, response) => {
+  response.send('Welcome to Destinys Golden Globe-API! Here is the endpoint for all the nominations https://goldenglobeawards.herokuapp.com/nominations, here is for getting a specific year and which won that year https://goldenglobeawards.herokuapp.com/nominations/years/2010?won=false etc')
 })
 
+// Get all the nominations and categories
+app.get('/nominations', (request, response) => {
+  response.json(goldenGlobesData)
+})
+
+app.get('/nominations/years/:year', (request, response) => {
+  const year = request.params.year
+  const showWon = request.query.won
+  let nominationsFromYear = goldenGlobesData.filter((item) => item.year_award === +year)
+
+  if (showWon) {
+    nominationsFromYear = nominationsFromYear.filter((item) => item.win)
+  }
+  response.json(nominationsFromYear)
+})
+
+// Tried to get a uniqe category back by using this code but instead a got back an empty array... 
+app.get('/nominations/categorys/:category', (request, response) => {
```

Comment:
For an endpoint that returns a single result it's a good idea to use the ID. Since this data doesn't have any ID's included - try to think what other element in the data comes the closest to being unique (e.g. nominee?) and change this endpoint to find that (filter returns an array as you mentioned) or add some ID's to your data